node.hcl

```diff
@@ -8,14 +8,15 @@ env = {
   "PATH": "${HERMIT_ENV}/node_modules/.bin:${NPM_CONFIG_PREFIX}/bin:${PATH}",
 }
 
-version "12.18.3" "12.22.6" "14.16.0" "14.17.0" "14.17.3" "16.9.0" "12.22.7"
-        "15.10.0" "14.18.0" {
+version "12.18.3" "12.22.6" "12.22.7" "12.22.12" "14.16.0" "14.17.0" "14.17.3" "14.18.0"
+        "14.19.0" "16.9.0" "15.10.0"  {
 }
 
+# arm64 only available for >=16
 version "16.1.0" "16.2.0" "16.3.0" "16.4.0" "16.5.0" "16.6.0" "16.6.1" "16.6.2"
         "16.7.0" "16.8.0" "16.9.1" "16.10.0" "16.11.0" "17.0.0" "17.0.1" "16.13.0" "17.1.0"
-        "16.13.1" "17.3.0" "17.3.1" "17.4.0" "14.19.0" "16.14.0" "17.5.0" "17.6.0" "17.7.0"
-        "17.7.1" "16.14.1" "17.7.2" "17.8.0" "12.22.12" "17.9.0" "18.0.0" {
+        "16.13.1" "17.3.0" "17.3.1" "17.4.0" "16.14.0" "17.5.0" "17.6.0" "17.7.0"
+        "17.7.1" "16.14.1" "17.7.2" "17.8.0" "17.9.0" "18.0.0" {
```

Comment:
One slight remaining issue here is that auto-versioning will randomly pick up old releases. I don't have a good solution though, and this is a great improvement already.